### PR TITLE
Remove old Foreman and Katello versions

### PR DESCRIPTION
**Pull request by sean797**

### CODE REVIEW

File: roles/foreman_repositories/tasks/redhat_release_repos.yml [hunk 1]
```diff
@@ -6,11 +6,3 @@
   tags:
     - packages
   when: foreman_repositories_version != 1.13 or ansible_distribution_major_version != "6"
```
Comment (ekohl) [inline roles/foreman_repositories/tasks/redhat_release_repos.yml]: This is redundant now.

File: config/base_boxes.yaml [hunk 2]
```diff
@@ -13,11 +13,6 @@ boxes:
     image_name: !ruby/regexp '/Debian.*Stretch/'
     pty:        true
 
-  centos6:
-    box_name:   'centos/6'
-    image_name: !ruby/regexp '/CentOS 6.*PV/'
-    pty:        true
```
Comment (ehelms) [inline config/base_boxes.yaml]: Technically this could be useful for client testing EL6.

File: config/base_boxes.yaml [hunk 3]
```diff
@@ -13,11 +13,6 @@ boxes:
     image_name: !ruby/regexp '/Debian.*Stretch/'
     pty:        true
 
-  centos6:
```
Comment (ekohl) [inline config/base_boxes.yaml]: It may be useful to keep this for a bit.

File: roles/katello_repositories/tasks/main.yml [hunk 4]
```diff
@@ -4,6 +4,3 @@
 
 - include_tasks: koji_repos.yml
   when: katello_repositories_use_koji
-
-- include_tasks: qpid.yml
-  when: ansible_distribution_major_version == "6" and katello_repositories_version == 3.2
```
Comment (ehelms) [inline roles/katello_repositories/tasks/main.yml]: Useful for client testing on EL6

**Comment by ehelms** [comment]
Looks generally good, I think I'd just like to keep the ability to client test EL6.

File: bats/fb-proxy-dns.bats [hunk 5]
```diff
@@ -144,13 +144,8 @@ setup() {
...
 
 @test "verify record type CNAME" {
-  if [[ $NAMESERVER == "bind" ]] ; then
-    (
-      [[ $FOREMAN_VERSION == "1.11" ]] ||
-      [[ $FOREMAN_VERSION == "1.12" ]] ||
-      [[ $FOREMAN_VERSION == "1.13" ]] ||
-      [[ $FOREMAN_VERSION == "1.14" ]]
-    ) && skip "CNAME records are only supported on 1.15+"
+  if [[ $NAMESERVER == "bind" ]] && [[ $(echo "$FOREMAN_VERSION < "1.15"" | bc) == 1 ]] ; then
```
Comment (ekohl) [inline bats/fb-proxy-dns.bats]: The quoting here looks weird

File: roles/katello_repositories/tasks/main.yml (hunk 4 above)
Comment (sean797) [reply roles/katello_repositories/tasks/main.yml]: Really? That `when` statement implies only Katello 3.2, which this PR removes.

Comment (ehelms) [reply roles/katello_repositories/tasks/main.yml]: Ahh, it was needed for 3.2 on RHEL6, but it is also required for a client on EL6.

**Comment by sean797** [comment]
Thanks for reviewing guys! Updated 👍

**Comment by ekohl** [comment]
Looks like 3.3 also fails so mind removing that too?

**Comment by sean797** [comment]
sure, updated.

**Comment by ekohl** [comment]
@ehelms any objections left?

**Pull request merged**